quickwit/quickwit-common/src/simple_list.rs

```diff
@@ -0,0 +1,83 @@
+// Copyright (C) 2022 Quickwit, Inc.
```

Comment:
Moved as it's used by elastic rest structs and quickwit one's.

Reply:
Maybe having a descriptive name for the file or putting in a module like `rest_api` can also help.